webapi/CopilotChat/Options/AzureFormRecognizerOptions.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.ComponentModel.DataAnnotations;
+using SemanticKernel.Service.Options;
+
+namespace SemanticKernel.Service.CopilotChat.Options;
+
+/// <summary>
+/// Configuration options for Azure Form Recognizer OCR support.
+/// </summary>
+public sealed class AzureFormRecognizerOptions
+{
+    public const string PropertyName = "AzureFormRecognizer";
+
+    /// <summary>
+    /// The endpoint for accessing a provisioned Azure Form Recognizer instance
+    /// </summary>
+    [Required, NotEmptyOrWhitespace]
```

Comment:
Won't this explode if we leave these strings empty because we set OCR type to none?

Reply:
Ah, no.  I think you're good.